prombench/manifests/prombench/benchmark/2a_prometheus-secret.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: s3-secret
+  namespace: prombench-{{ .PR_NUMBER }}  # Replace with your actual namespace
```

Comment:
I see you put the Secret in the namespace for the PR, but I see the same bucket being used across many runs.